hack/lib/golang.sh

```diff
@@ -101,7 +101,8 @@ kube::golang::server_image_targets() {
 
 IFS=" " read -ra KUBE_SERVER_IMAGE_TARGETS <<< "$(kube::golang::server_image_targets)"
 readonly KUBE_SERVER_IMAGE_TARGETS
-readonly KUBE_SERVER_IMAGE_BINARIES=("${KUBE_SERVER_IMAGE_TARGETS[@]##*/}")
+readonly KUBE_SERVER_LINUX_IMAGE_BINARIES=("${KUBE_SERVER_IMAGE_TARGETS[@]##*/}")
+readonly KUBE_SERVER_WINDOWS_IMAGE_BINARIES=("kube-proxy.exe")
```

Comment:
hard-coding specific binaries and platforms into this script doesn't seem correct ... shouldn't these be defined alongside KUBE_SERVER_IMAGE_TARGETS, not here?

Reply:
also, can we leave KUBE_SERVER_IMAGE_BINARIES as-is and add a KUBE_SERVER_WINDOWS_IMAGE_BINARIES, rather than renaming the existing variable? it's a lot harder to reason about impact of changes that are not strictly additive